shared/copyq.iss

```diff
@@ -53,7 +53,7 @@ Name: cz; MessagesFile: "compiler:Languages\Czech.isl"
 Name: de; MessagesFile: "compiler:Languages\German.isl"
 Name: es; MessagesFile: "compiler:Languages\Spanish.isl"
 Name: it; MessagesFile: "compiler:Languages\Italian.isl"
-Name: ko; MessagesFile: "{#Output}\Korean.isl"
+Name: ko; MessagesFile: "compiler:Languages\Korean.isl"
```

Comment:
Setting this won't work unless the Inno Setup contains the file. I tried to download it to the Inno Setup "Languages" folder in #2315 but it did not work. I assume Inno Setup requires other changes to include an unofficial language.